llvm/test/Transforms/InstCombine/sub.ll

```diff
@@ -2626,3 +2626,141 @@ define i8 @sub_of_adds_2xc(i8 %x, i8 %y) {
   %r = sub i8 %xc, %yc
   ret i8 %r
 }
+
+define i8 @test_neg_of_udiv_of_nonnegs(i8 %a) {
```

Comment:
```suggestion
define i8 @test_neg_of_udiv_of_nonnegs_nuw(i8 %a) {
```
Was very confused by this test until I realized this... TBH I think it can be omitted entirely, as it doesn't really have any relation to this fold.